src/frontend/apps/standalone_site/src/features/Contents/hooks/useContentRoutes.tsx

```diff
@@ -0,0 +1,57 @@
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { ReactComponent as VueListIcon } from 'assets/svg/iko_vuelistesvg.svg';
+import { LoadSVG } from 'components/Assets';
+import { RouteRequired } from 'routes';
+
+import { useContentFeatures } from '../store/contentsStore';
+
+const messages = defineMessages({
+  menuContentsLessonsLabel: {
+    defaultMessage: 'Lessons',
+    description: 'Label for the Lessons link in the content navigation menu',
+    id: 'Contents.useContentRoutes.menuContentsLessonsLabel',
+  },
```

Comment:
This can be removed IMO./ Lessons does not exists in marsha. I think we are keeping it since the first prototype we made

Reply:
[🔥(website) remove code about lessons](https://github.com/openfun/marsha/pull/2253/commits/581d5cf80434f0acd45f622d234c5f2fbfa43a34)